packages/testing/src/api/index.ts

```diff
@@ -134,7 +134,7 @@ function initializeApp(
         ({
           getToken: async () => ({ accessToken: accessToken }),
           getUid: () => null,
-          addAuthTokenListener: (listener) => {
+          addAuthTokenListener: listener => {
```

Comment:
I don't see why we need this change, but if it's an auto-formatting or linter thing, then you have my approval.

Reply:
Yeah, this is from auto-formatting.